components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -106,24 +107,121 @@ chef-alert {
     font-size: 14px;
   }
 
-  .health-line {
-    display: flex;
+  hr.health-info-divider {
+    margin: 16px -1em 16px -1em;
+    background-color: $gray-pale;
+  }
+
+  .health-info-panel {
+    border-radius: 2px;
     align-items: center;
+    padding: 8px;
     font-size: 12px;
-    margin-bottom: 6px;
+
+    &.CRITICAL {
+      background-color: var(--chef-badge-critical-background);
+      color: var(--chef-badge-critical-color);
+    }
+
+    &.WARNING {
+      background-color: var(--chef-badge-warning-background);
+      color: var(--chef-badge-warning-color);
+    }
+
+    &.UNKNOWN {
+      background-color: var(--chef-lightest-grey);
+      color: var(--chef-primary-dark);
+    }
+
+    &.OK {
+      background-color: var(--chef-badge-success-background);
+      color: var(--chef-badge-success-color);
+    }
+
   }
 
-  .channel-line {
-    padding-left: 18px;
+  .disconnect-notice {
```

Comment:
Most of this isn't new, I moved it to be as nested as the other things so this file would be consistent with itself.